amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/UpdateField.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.kinesis.leases;
+
+public enum UpdateField {
+    CHILD_SHARDS, HASH_KEY_RANGE
+}
```

Comment:
Should we add a blacklist of fields that shouldn't be updated in this way (would interfere with normal lease management operations)? Or a least a comment?